buffer_driver.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+EMPTY_BUFFER = "empty"
+BUFFER_INDEX_START = 1
+BUFFER_INDEX_END = 5
+BUFFER_FILE_PARAMETERS = 4
```

Comment:
이거는 파라미터 개수를 의미하나요?
```suggestion
NUM_BUFFER_FILE_PARAMETERS = 4
```
그렇다면 이런건 어떠신지요